Dockerfile

```diff
@@ -46,20 +46,22 @@ RUN useradd -m --shell /bin/false app
 RUN echo "locales locales/default_environment_locale select en_US.UTF-8" | debconf-set-selections
 RUN echo "locales locales/locales_to_be_generated multiselect en_US.UTF-8 UTF-8" | debconf-set-selections
 
+# Setting up ppa deadsnakes to get python 3.11
+RUN apt-get update && \
+  apt-get install -y software-properties-common && \
+  apt-add-repository -y ppa:deadsnakes/ppa
+
 # Install requirements that are absolutely necessary
 RUN apt-get update && \
     apt-get -y dist-upgrade && \
     apt-get -y install --no-install-recommends \
-        python3 \
-        python3-venv \
-        python3.8 \
-        python3.8-minimal \
-        # python3-dev: required for building mysqlclient python package version 2.2.0
-        python3-dev \
-        libpython3.8 \
-        libpython3.8-stdlib \
+        python3-pip \
+        python3.11 \
+        python3.11-dev \
+        python3.11-venv \
+        libpython3.11 \
+        libpython3.11-stdlib \
         libmysqlclient21 \
-        # libmysqlclient-dev: required for building mysqlclient python package version 2.2.0
```

Comment:
Are these comments no longer relevant? If so, are the `-dev` packages themselves no longer needed?

Reply:
We've already surpassed the mysqlclient version 2.2.0 in our requirements as the latest PyPI version is now 2.2.4 instead. 
The dev package is still needed but I've removed the comment because it mentioned an outdated version of mysqlclient.

Reply:
I think it would be best to keep the comment but just remove the specific version mention.